x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/EnrollNodeToClusterTests.java

```diff
@@ -231,6 +247,29 @@ public void testEnrollmentExitsOnExistingSecurityConfiguration() throws Exceptio
         assertAutoConfigurationFilesNotCreated();
     }
 
+    @SuppressForbidden(reason = "Cannot use getDataPath() as Paths.get() throws UnsupportedOperationException for jimfs")
```

Comment:
These suppress forbiddens are too broad. I see there are other ones in this file too. Please isolate the call that is forbidden in its own method so other forbidden uses don't creep in.

Reply:
will amend